components/authz-service/habitat/config/config.yml

```diff
@@ -17,3 +17,5 @@ data-migrations-path: {{ pkg.svc_static_path }}/data-migrations
 event-address: {{event-service.sys.ip}}:{{event-service.cfg.port}}
   {{~/if}}
 {{~/eachAlive}}
+
+config: {{pkg.svc_data_path}}/.authz-service.toml
```

Comment:
Adding the config file to the services data directory. `/hab/svc/authz-service/data/.authz-service.toml`